packages/build/src/services/bundler/vendor.ts

```diff
@@ -11,13 +11,41 @@ export interface VendorBundleServiceEvents extends OutputServiceEvents {
   [BundleVendorErrorEvent]: () => void
 }
 
+const includesDefault = [
+  'react',
+  'react-dom',
+  'react-helmet',
+  'lodash',
+  'typestyle',
+]
+
 export class VendorBundleService {
+  static get includes() {
```

Comment:
how about moving this to a lib reading our project/package definition
could be in that file or the folder: 
https://github.com/seagull-js/seagull/blob/master/packages/build/src/lib/project.ts

Reply:
sure, didn't find a good spot to put it, so i left it in the class for now.